pkg/anonymiser/anonymiser_test.go

```diff
@@ -106,6 +112,22 @@ func testWhenColumnIsAnonymisedWithLiteral(t *testing.T, opts reader.ReadTableOp
 	}
 }
 
+func testWhenColumnAnonymiserIsInvalid(t *testing.T, opts reader.ReadTableOpt, tables config.Tables) {
+	anonymiser := NewAnonymiser(&mockReader{}, tables)
+
+	rowChan := make(chan database.Row)
+	defer close(rowChan)
+
+	err := anonymiser.ReadTable("test", rowChan, opts)
+	require.NoError(t, err)
+
+	for {
+		row := <-rowChan
```

Comment:
Can we have a timeout in this to avoid having a test run for ever?

Reply:
Done